<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alterar_Notas</class>
 <widget class="QDialog" name="Alterar_Notas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Matricula</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pesquisa_Aluno_Nota"/>
   <widget class="QPushButton" name="Btn_PesquisaNota">
    <property name="text">
     <string>Pesquisar</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="Tv_AlunoNota">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>521</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mat√©ria</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nota 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Nota1Inserir"/>
      <widget class="QPushButton" name="Btn_AlterarNota1">
       <property name="text">
        <string>Inserir/Alterar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Nota 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="Nota2Inserir"/>
      <widget class="QPushButton" name="Btn_AlterarNota2">
       <property name="text">
        <string>Inserir/Alterar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Nota 3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="Nota3Inserir"/>
       <widget class="QPushButton" name="Btn_AlterarNota3">
        <property name="text">
         <string>Inserir/Alterar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_VoltarProfessorAlterNota">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>voltar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
